features/ownership/permission.feature

```diff
@@ -90,4 +90,26 @@ Feature: Test permissions based on ownership
         {"name": "madewithlove"}
     """
     And I send a PUT request to "/events/%{eventId}/name/nl"
+    Then the response status should be "403"
+
+  Scenario: Approving the ownership of an organizer gives permission no permission on the place associated with the organizer
```

Comment:
```suggestion
  Scenario: Approving the ownership of an organizer doesn't give permission on the place associated with the organizer
```
`gives permission no permission ` is a bit difficult to understand if you are not familiar with the background.

Reply:
That was indeed a copy/paste error